src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -33,6 +35,7 @@ import { SpendingTypesTT } from 'components/covid19/Covid19Tooltips';
 
 const propTypes = {
     type: PropTypes.string.isRequired,
+    pageDefCodes: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
It looks like this variable gets assigned on line 286 rather than passed down as a prop, so I think we can remove this line

Reply:
Yep, I tried to make it a prop, didn't work.